demos/chips.html

```diff
@@ -281,8 +285,17 @@ <h2>Custom theme</h2>
           root && inputChipSetEl.removeChild(root);
         }
 
+        function deleteLastChip(evt) {
+          if (evt.type === 'click' || evt.key === 'Enter' || evt.keyCode === 13) {
```

Comment:
This `if` is unnecessary; this function is only used in response to `click` which already triggers on enter/space for buttons.